src/main/java/net/homework/webapp/service/convert/EmployeeDtoService.java

```diff
@@ -0,0 +1,44 @@
+package net.homework.webapp.service.convert;
+
+import net.homework.webapp.dto.EmployeeDto;
+import net.homework.webapp.model.Department;
+import net.homework.webapp.model.Employee;
+import net.homework.webapp.repository.DepartmentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmployeeDtoService {
+
+    @Autowired
+    private DepartmentRepository departmentRepository;
```

Comment:
mapper should map only, not to query in db

Reply:
fixed